### PR TITLE
Tests for `metrics()`

### PR DESCRIPTION
**Pull request by vtols**
Can you suggest additional test cases?

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/clause/where/WhereInMetricsTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    public static void prepareData() throws Exception {
+        Registry.Entity.register(ENTITY_NAME);
+
+        List<Series> seriesList = new ArrayList<>();
+
+        for (int i = 0; i < METRIC_COUNT; i++) {
+            METRIC_NAMES[i] = metric();
+            Registry.Metric.register(METRIC_NAMES[i]);
+
+            Series series = new Series();
```
Comment (VeselovAlex) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/where/WhereInMetricsTest.java]: Why not to use Mocks.series() ?

File: src/test/java/com/axibase/tsd/api/method/sql/clause/where/WhereInMetricsTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        SeriesMethod.insertSeriesCheck(seriesList);
+    }
+
+    /**
+     * #4075
+     */
+    @Test
+    public void testMetricsBasic() {
+        String sqlQuery = String.format(
+                "SELECT metric FROM atsd_series WHERE metric IN metrics('%s') ORDER BY metric",
```
Comment (VeselovAlex) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/where/WhereInMetricsTest.java]: Better split into line-per-clause

File: src/test/java/com/axibase/tsd/api/method/sql/clause/where/WhereInMetricsTest.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                ENTITY_NAME
+        );
+
+        String[][] expectedRows = new String[METRIC_COUNT][];
+        for (int i = 0; i < METRIC_COUNT; i++)
```
Comment (VeselovAlex) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/where/WhereInMetricsTest.java]: Pls add braces

File: src/test/java/com/axibase/tsd/api/method/sql/clause/where/WhereInMetricsTest.java (hunk 1 above)
Comment (vtols) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/where/WhereInMetricsTest.java]: Why `Mocks.series()` is better than `new Series()`

**Pull request merged**